File: ArrayAndString/344. Reverse String.java
class Solution {
    public void reverseString(char[] s) {
        char temp = 'a';
        int start = 0;
        int end = s.length-1;
        while(start < end){
            temp = s[start];
            s[start] = s[end];
            s[end] = temp;
            start++;
            end--;
        }
        
    }
}

File: ArrayAndString/344. Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        length = len(s)
        for x in range((length+1) //2):
            temp = s[x]
            s[x] = s[length-1-x]
            s[length-1-x] = temp
            
            
        """
        Do not return anything, modify s in-place instead.
        """
        